src/main/resources/story/TrelloLogin/TrelloLogin.story

```diff
@@ -1,30 +1,38 @@
 Description: checking login homepage for trello.com
 
+
+!-- Positive scenarios : login and logout flow
 Scenario: login to trello home page with valid credentials 
```

Comment:
Login functionality will be used in other stories. Please make it a composite step